functions.php

```diff
@@ -0,0 +1,15 @@
+<?php
+function displayCharacters(array $allCharacters): string {
```

Comment:
you might want to rename this to something that describes what it does more semantically, eg generateCharacterHtml